### ISSUE TITLE
[Feat]: Custom PR body

### ISSUE THREAD

**Issue by kdeldycke** [AndreasAugustin/actions-template-sync#477]
### Describe the feature

Current PR body looks like this:

![Screenshot 2024-03-02 at 11 24 42](https://github.com/AndreasAugustin/actions-template-sync/assets/159718/e0f11897-0d0c-423c-b3b0-0551b6ed107e)

I'd love to have the possibility of feeding the `actions-template-sync` with a custom parameter like `body`:
```yaml
      - name: actions-template-sync
        uses: AndreasAugustin/actions-template-sync@v1.8.0
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          source_repo_path: "kdeldycke/awesome-template"
          body: >
            ### Description


            Ready to be merged into `main` branch, at the discretion of the maintainers, to synchronize the content
            with the reference [`awesome-template` repository](https://github.com/kdeldycke/awesome-template).


            ---


            <details><summary><code>Workflow metadata</code></summary>


            > [Auto-generated on run `#${{ github.run_id }}`](${{ github.event.repository.html_url }}/actions/runs/${{
            github.run_id }}) by `${{ github.job }}` job from [`docs.yaml`](${{ github.event.repository.html_url
            }}/blob/${{ github.sha }}/.github/workflows/docs.yaml) workflow.


            </details>
```

This is inspired by [`peter-evans/create-pull-request` action](https://github.com/peter-evans/create-pull-request) which supports a `body` parameter.

### Use Case

A custom body will allow me to add context to the PR, like direct links to the source repository and commit, so a maintainer can inspect the PR before deciding to merge it or not.

Also: emoji! 🥳

### Acknowledgements

- [ ] I may be able to implement this feature request

**Comment by AndreasAugustin** [comment]
@kdeldycke thanks for the feature request. Available with `v1.9.0`

**Comment by kdeldycke** [comment]
I can confirm this feature is working! Thanks @AndreasAugustin for adding it!
